tests/platform/windows/windows-jaws-testSpec.txt

```diff
@@ -5,6 +5,6 @@ This will run the acceptance tests for the JAWS screenreader (Freedom Scientific
 
 Prerequisites:
 * JAWS needs to be installed
-* In JAWS, a profile should be created by going to "options"->voices->Voice Adjustment. Make sure that "Eloquence - (factory)" is selected in the profile name. Click "Save As.." button and write "cloud4all" as the name (no quotes). Once this is done, Jaws should be working properly
+* In JAWS, a profile should be created by going to "options"->voices->Voice Adjustment. Make sure that "Eloquence - (factory)" is selected in the profile name. Click "Save As.." button and write "GPII" as the name (no quotes). Once this is done, Jaws should be working properly
 
```

Comment:
Yes, there's a way to do this automatically.

JAWS has two settings file: the main settings file "DEFAULT.jcf" and the voice profile setting "xxx.JCF". We just need to change the Voice "Profiles.ActiveVoiceProfileName = xxx" setting in "DEFAULT.jcf". "xxx" will be the name of the voice profile setting.

Currently, I'm setting the Voice "Profiles.ActiveVoiceProfileName = GPII"and naming the voice profile "GPII.vpf". You will also have to set this setting in "Options.PrimarySynthesizer = eloq" in the voice profile settings.

The problem that I'm facing now is that GPII won't create the settings file if it does not exist. Here's the error I got : {"isError":true,"message":"ENOENT, no such file or directory 'C:\\Users\\user\\AppData\\Roaming\\Freedom Scientific\\JAWS\\17.0\\Settings\\VoiceProfiles\\GPII.VPF'"}
I think this can be solved by adding a few lines of code in the main settings handler code to allow file creation if it does not exist.

There might be other alternative ways. JAWS has an API and I'm looking into it but it requires running a script.